src/azure-data-transfer/README.md

```diff
@@ -0,0 +1,241 @@
+# Azure CLI automation Extension #
+This package is for the 'azure data transfer' extension, i.e. 'az azure-data-transfer'.
+
+### How to use ###
+Install this extension using the below CLI command
+```
+az extension add --name azure-data-transfer
+```
+
+### Included Features
+#### Viewing Pipelines:
+Manage Connection: [more info](https://learn.microsoft.com/en-us/azure/templates/microsoft.azuredatatransfer/pipelines/)\
+*Examples:*
+
+##### Get a pipeline
+
+```
+az azure-data-transfer pipeline show \
```

Comment:
do you think `az azure-data-transfer xx` is a good name? cauz, `az` and `azure-*` have the duplicate meaning here.